### PR TITLE
Add deleting private rules, add creating private rules from SCP and command line

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/writing-rules/private-rules.md [hunk 1]
```diff
@@ -7,21 +7,41 @@ import MoreHelp from "/src/components/MoreHelp"
...
+Users of the [Team or Enterprise tier](https://semgrep.dev/pricing) of Semgrep Code can publish rules to the [Semgrep Registry](https://semgrep.dev/explore) as private rules that are not visible to others outside their organization. The private rules enable you to hide code-sensitive information or legal requirements that prevent you from using a public registry.
 
 As we continue to develop and refine this feature, we welcome and appreciate your feedback! Email us at [product@r2c.dev](mailto:product@r2c.dev) or contact us in the [Semgrep Community Slack](https://go.semgrep.dev/slack).
 
-## Getting started
+## Creating private rules
 
-To create private rules, use the [Semgrep CLI](../getting-started.md) to run these commands:
+Create private rules the same way you create other custom rules. Private rules are stored in Semgrep Registry but are not visible outside of your organization. Two sections below can help you to create and save your private rules either through Semgrep Cloud Platform or through your command line.
```
Comment (adamkvitek) [inline docs/writing-rules/private-rules.md:16]:
```suggestion
Create private rules the same way you create other custom rules. Private rules are stored in Semgrep Registry but they are not visible outside your organization. The two sections below can help you to create and save your private rules.
```

File: docs/writing-rules/private-rules.md [hunk 2]
```diff
@@ -7,21 +7,41 @@ import MoreHelp from "/src/components/MoreHelp"
...
+
+1. Go to [Semgrep Editor](https://semgrep.dev/orgs/-/editor).
+1. Click <i className="fa-solid fa-file-plus-minus inline_svg"></i> **Create New Rule**.
+1. Choose one of the following:
+    - Create a new rule and test code by clicking <i class="fa-solid fa-circle-plus"></i> **plus** icon, select **New rule**, and then click <i className="fa-solid fa-floppy-disk inline_svg"></i> **Save**. Note: The test file must contain at least one true positive and one true negative test case to be approved. See the [Tests](#tests) section of this document for more information.
+    - In the <i class="fa-solid fa-server"></i> **Library** panel, select a rule from a category in **Semgrep Registry**. Click <i className="fa-solid fa-code-branch inline_svg"></i> **Fork**, modify the rule or test code, and then click <i className="fa-solid fa-floppy-disk inline_svg"></i> **Save**.
+
+Your private rule has been created. Private rules are stored in the folder with the same name as your Semgrep Cloud Platform organization.
+
+### Creating private rules through command line
```
Comment (s-santillan) [inline docs/writing-rules/private-rules.md:34]:
```suggestion
### Creating private rules through the command line
```

File: docs/writing-rules/private-rules.md [hunk 3]
```diff
@@ -7,21 +7,41 @@ import MoreHelp from "/src/components/MoreHelp"
...
+To create private rules through the [Semgrep CLI](/getting-started/), :
 
 1. Interactively login to Semgrep:
 
     ```sh
     semgrep login
     ```
-
-2. Run `semgrep publish` followed by the path to your private rules:
+1. Create your rule and test code. For more information, see [Contributing rules](/contributing/contributing-to-semgrep-rules-repository/) documentation.
```
Comment (s-santillan) [inline docs/writing-rules/private-rules.md:43]: Do folks creating private rules actually need complete test cases, or is this optional? AFAIK I can just write a rule with no test cases.

**Review by s-santillan — APPROVED** [review]
lgtm!

File: docs/writing-rules/private-rules.md (hunk 3 above)
Comment (adamkvitek) [reply docs/writing-rules/private-rules.md:43]: @s-santillan We were always recommended to submit the test code also (but I need to test if you can submit both the test code and the rule with the command below this procedure).

@ThatDevShparki maybe you know? :-) Please check this document.

Comment (ThatDevShparki) [reply docs/writing-rules/private-rules.md:43]: I don't believe you have to submit tests! Though the rule itself does have to be valid and succeed on a single run to be saved

**Review by ThatDevShparki — APPROVED** [review]
Everything looks good to me!

File: docs/writing-rules/private-rules.md (hunk 3 above)
Comment (adamkvitek) [reply docs/writing-rules/private-rules.md:43]:
```suggestion
1. Create your rule. For more information, see [Contributing rules](/contributing/contributing-to-semgrep-rules-repository/) documentation.
```

**Pull request merged**